.github/workflows/cbmc-latest.yml

```diff
@@ -44,34 +67,21 @@ jobs:
           cmake -S . -Bbuild -DWITH_JBMC=OFF
           cmake --build build -- -j 4
           sudo cmake --build build --target install
-      - name: Execute Kani regressions
-        run: ./scripts/kani-regression.sh
+          # Cleanup cbmc directory
+          cd ..
+          rm -rf cbmc
 
-  perf:
-    runs-on: ubuntu-20.04
-    steps:
       - name: Checkout Kani
         uses: actions/checkout@v3
 
       - name: Setup Kani Dependencies
         uses: ./.github/actions/setup
         with:
           os: ubuntu-20.04
+          install_cbmc: 'false'
```

Comment:
Are you sure this works? I've seen cases where inputs and outputs are not converted to booleans correctly when used in expressions like `if: ${{ inputs.install_cbmc }}`. Since `'false'` is a non-empty string, it evaluates to `true`.

The [ubuntu-20.04 run](https://github.com/model-checking/kani/actions/runs/3942005640/jobs/6745140138#step:3:5) for this PR does indeed seem to set `install_cbmc` to true... 

Suggestion [here](https://github.com/actions/runner/issues/1483#issuecomment-996183227) is to do:
```
if: ${{format('{0}', inputs.install_cbmc) != 'false'}}
```

Reply:
Indeed, it failed! Thanks for pointing out the fix. I'll apply your suggestion.

Reply:
I ran the workflow in my fork and confirmed that the CBMC installation was skipped:

https://github.com/zhassan-aws/kani/actions/runs/3942199886/jobs/6745566157